### DISCUSSION TITLE
Slingbox 500 Questions

### DISCUSSION THREAD

**Discussion by cgcmgr** [GerryDazoo/Slinger#666] [General]
Hey everyone,

Just a few questions regarding the Slingbox 500.  Currently I have the M1 and I'm using it in conjunction with Gerry's Slinger and SlingerPlayer.  It works great, thanks for this Gerry!!  I also had a ProHD, but a popping noise started, so I no longer use it.

I'm think of switching to a Slingbox 500 for connectivity reasons.  The new TV I want to get for my bedroom does not have Component inputs and I don't want to use a Component to HDMI convertor.

I did a lot of reading in this forum, but just want to make sure a few scenarios are correct.

1. If I buy a brand new never opened Slingbox 500, I can still use it with the admin password of 'admin' and by following this video - https://www.youtube.com/watch?v=EUyQfpzHRkg.  The only drawback is, it will never finish the setup due to not being able to connect to the Sling servers.  The setup screen will be on the TV all the time.  It will work with with Slinger and SlingerPlayer though.  My questions is, will the remote I have in my Config file work to change channels?
2. If I buy a used Slingbox 500 and do not have the admin password, I can reset it and pretty much have the same experience as number 1 above.
3. If I buy a used Slingbox 500 and the seller is willing to check the Console.log file here - C:\Users________\AppData\Roaming\Sling Media\Slingplayer Desktop\Logs, and provide me with the admin password and port, I should be able to use that info in my Config file for Slinger, assuming the IP isn't static and is set to DHCP.  Again, would I be able to use the remote I have in my Config file work to change channels?

I guess my main question is, will the remote work in any of the above scenarios?  I'm not talking about a physical remote, I'm talking about the remote in the config file that you access via the browser.  I access my Slingbox remotely and that's the only remote I need as it serves me well.  Currently, my Config file is using the FIOS.txt remote.  Will this work, or does the Slingbox still need a remote BIN file loaded?

Thanks,
Chris

**Comment by GerryDazoo** [comment]
To answer your main question. The remote will not work. The Slingbox takes the info in the .bin file and uses it to map keycodes (what's in the remote.txt file ) to the actual IR signals needed for your cable box.
Try to configure the remote with a predefined bin file from Sling will not work. But some have reported that using a 3rd party bin file has been successfully install. YMMV. I have not confirmed this because I don't want to mess up my working boxes.
I believe there are other external boxes that can be used for remote control operation.

**Reply by cgcmgr** [reply]
Thanks Gerry.  That's what I figured.  
So I can either try loading a custom BIN file from here - http://www.hifi-remote.com/forums/dload.php?action=category&cat_id=87 
using this tutorial - http://www.hifi-remote.com/wiki/index.php/How_to_Install_a_Custom_Remote_onto_the_Slingbox.  
My box is an Arris, I have Verizon FIOS, or get lucky and find a used Slingbox 500 that has the Arris Bin file already installed in it, which is pretty much a crap shoot.

Thanks for the help.

**Reply by cgcmgr** [reply]
UPDATE:

Maybe this tutorial isn't the one I should be following, http://www.hifi-remote.com/wiki/index.php/How_to_Install_a_Custom_Remote_onto_the_Slingbox.  

It seems as if I would need to access newwatch.slingbox.com which doesn't seem to be accessible anymore.  Would you happen to know what process others are using to install a 3rd party BIN file?  Is there a "tutorial" of 
sorts or a thread that explains it anywhere that you're aware of?

Chris

**Reply by p312-hub** [reply]
Were you ever able to figure this out? Stuck with a 500 without remote control, and the above links are look to be dead

**Reply by RayMan412** [reply]
I just emailed the website help...see if its completely done..

**Reply by RayMan412** [reply]
And this is likely your best option. Yes it's an additional piece....but it does work.
https://www.amazon.com/Broadlink-RM4-Universal-Control-Compatible/dp/B07ZSF46BX/ref=sr_1_9?crid=2Q2HBL0CZQ8AA&dib=eyJ2IjoiMSJ9.FJS_W4ibFV5BitDb4ZpDRVwLrlV5Acu8mAeqS8DNVj3D9859a8Unmgr-11U24Mw5tlO95z1ZX2_8tKpaoWGQ6u40kUNq7eFXM7grj3487sZg7-sk16eJLxVxhvQpG4-Ggu62qYbeKFOsj7sx7MxerggCdnEfiASC_9BvlMV-RHV-ADI9FljJbZqYaryGP5kZpAfEr929gK-lMjgP_3kJurBCmF8MKyv8y7WmHEOPm5o.9ihfMYAPzsHLqq2HNoDvIa1M4GWV-9jgOQgz-XXMw-Y&dib_tag=se&keywords=IR%2Bnetwork%2Bremote&qid=1756139882&sprefix=ir%2Bnetwork%2Bremote%2Caps%2C114&sr=8-9&th=1

**Reply by GerryDazoo** [reply]
Yes, I ended up getting RM4Mini works well from anywhere on Internet. I'm not going to invest the time to try and reverse engineer adding third party bins to the Slingbox.

**Reply by RayMan412** [reply]
You had mentioned possibly adding that RM4Mini to the slinger remote interface. Is that something still possible? Not needed just yet....but it could be when my cable cards die.

**Reply by GerryDazoo** [reply]
Thanks for pinging me on this. I had filled out the forms to get access to their API but i did not hear back from them. I guess a few remotes for old slingboxes isn't big enough market to waste time on me. I'll followup again.

**Comment by RayMan412** [comment]
Same Issue I have here. I have three working M2,500s and 350s. All pre configured. You can't seem to access their remote settings. The ProHD i was able to resurrect and now with my MAC I can even scan the digital. It's been great. If we can find some way to get the remote in the 350/500/M1-2 then we are golden. It does appear that the shutdown of the servers have stopped the bricking, but I'm not 100% sure on that. My ProHd isn't fully protected as my router is strange but it's been working great.

**Reply by RayMan412** [reply]
But the 20/30 dollar broadlink remote is a very good easy fix.

**Comment by GerryDazoo** [comment]
I found a little project on github (https://github.com/mjg59/python-broadlink) that implements a simple broadlink control interface in python. So, I don't need the broadlink API code. I've hooked it into my latest code and it appears to be working. Building the keycode to IR data mapping file is a pain so I might write a little utility to make it easier. Let me know if you want to take it out for a test drive.
It will work over the Internet with the proper port forwarding rules in your router where the broadlink/slingbox are located.
